testing/build.gradle

```diff
@@ -1,7 +1,7 @@
 description = "gRPC: Testing"
+
 dependencies {
-    compile project(':grpc-core'),
-            project(':grpc-stub'),
+    compile project(':grpc-testing-proto'),
```

Comment:
The grpc-testing-proto dependency should be a `testCompile` dependency (vs `compile`).

(That'll force you to add testing-proto as a new dep to interop-testing.)